vars/buildDotNetCore.groovy

```diff
@@ -0,0 +1,75 @@
+def call(Map config=[:], Closure body={}) {
+  def containerTag = ''
+  def mergedPrNo = ''
+  def pr = ''
+  def repoName = ''
+
+  node {
+    checkout scm
+    try {
+      stage('Set GitHub status as pending') {
+        build.setGithubStatusPending()
+      }
+      stage('Set PR, and containerTag variables') {
+        (repoName, pr, containerTag, mergedPrNo) = build.getVariables(version.getCSProjVersion(config.project))
+      }
+      stage('Helm lint') {
+        test.lintHelm(repoName)
+      }
+      stage('Build test image') {
+        build.buildTestImage(DOCKER_REGISTRY_CREDENTIALS_ID, DOCKER_REGISTRY, repoName, BUILD_NUMBER)
+      }
+      stage('Run tests') {
+        build.runTests(repoName, repoName, BUILD_NUMBER)
+      }
+      stage('Push container image') {
+        build.buildAndPushContainerImage(DOCKER_REGISTRY_CREDENTIALS_ID, DOCKER_REGISTRY, repoName, containerTag)
+      }
+
+      if (pr != '') {
+        stage('Verify version incremented') {
+          version.verifyCSProjIncremented(config.project)
+        }
+        stage('Helm install') {
+          helm.deployChart(config.environment, DOCKER_REGISTRY, repoName, containerTag)
+          echo "Build available for review at https://ffc-demo-$containerTag.$INGRESS_SERVER"
```

Comment:
This echo line should be removed as not all services will have ingress and those that do won't be at this address